kqueen_ui/server.py

```diff
@@ -25,7 +22,7 @@ def create_app(config_file=None):
     app.register_blueprint(ui, url_prefix='/ui')
 
     # load configuration
-    config = current_config(config_file)
+    config = config_file or kqueen_ui_config
```

Comment:
user have no way to pass config_file here, so I suggest to remove this parameter at all. Need to know Adam opinion